.expeditor/scripts/verify/run_cargo_test.sh

```diff
@@ -2,8 +2,7 @@
 
 set -eou pipefail
 
-# shellcheck source=./support/ci/shared.sh
-source ./support/ci/shared.sh
+source .expeditor/scripts/verify/shared.sh
```

Comment:
Is it needed? shellcheck passes without it so i'm not sure what it gains us in this instance.

Reply:
It improves shellcheck's analysis by letting it access the sourced file. This can be used to resolving things like undefined variables and the like. Ideally we'd use it everywhere we source static files, but that's unrelated to this PR. I just don't think we should remove it since it does add value.